tests/unit/test_dimension.py

```diff
@@ -442,10 +442,12 @@ def it_resolves_a_categorical_type_to_help(
         is_logical_type,
         expected_value,
         _is_array_cat_prop_,
+        _is_cat_date_prop_,
         _has_selected_category_prop_,
         _is_logical_type_prop_,
     ):
         _is_array_cat_prop_.return_value = is_array_cat
+        _is_cat_date_prop_.return_value = False
```

Comment:
Because I don't want it to enter the function code, because then it'll crash. The crash is because the iterable is initialized to `None` from the unit test. But we don't need to cover that case in the real world (iterable being set to `None`), so I've taken that part of the code out. And now it's setting the unit test in a more realistic way. Here's the specific change that this allows:

https://github.com/Crunch-io/crunch-cube/pull/346/files#diff-84a1fae6b8f21e7c557dfbf8ea91f3306a10a3cb9e704b279f29d3ab6d915964L254